packages/util/env-defaults

```diff
@@ -43,7 +43,7 @@ USE_LOCALSTACK=true
 LOCALSTACK_HOST=localstack
 
 # Identifier so backends know which Harmony client submitted the request
-CLIENT_ID=harmony-local
+CLIENT_ID=harmony-unknown
```

Comment:
It might make sense to set `CLIENT_ID=harmony-in-a-box` in the create-dotenv script.